script/gn-check.js

```diff
@@ -0,0 +1,25 @@
+const sh = require('shelljs')
+const path = require('path')
+const fs = require('fs')
+
+const SRC_DIR = path.resolve(__dirname, '..', '..')
+let GN_CHECK_DIR
+
+if (process.env.ELECTRON_OUT_DIR) {
+  GN_CHECK_DIR = `../out/${process.env.ELECTRON_OUT_DIR}`
+} else {
+  for (const buildType of ['Debug', 'Testing', 'Release']) {
+    const outPath = path.resolve(SRC_DIR, 'out', buildType)
+    if (fs.existsSync(outPath)) {
+      GN_CHECK_DIR = `../out/${buildType}`
+      break
+    }
+  }
+}
+
+if (!GN_CHECK_DIR) {
+  throw new Error(`No viable out dir: one of Debug, Testing, or Release must exist.`)
+}
+
+const gnCheckString = `gn check ${GN_CHECK_DIR} //electron:electron_lib`
```

Comment:
We should run the same checks we do in CI:
https://github.com/electron/electron/blob/master/.circleci/config.yml#L264

Reply:
ooh good point, will do!